src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,19 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  return (
+    <article
+      className={completed
+        ? 'TodoInfo TodoInfo--completed'
```

Comment:
The ternary operator is not necessary here. You can simplify this by always including the 'TodoInfo' class and conditionally adding the 'TodoInfo--completed' class if the todo is completed.

Reply:
thanks, done